### PR TITLE
Add document for sealwrap/rewrap endpoint

### PR DESCRIPTION
**Pull request by mgaffney**

### CODE REVIEW

File: website/source/api/system/sealwrap-rewrap.html.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+### Sample Response
+
+```json
+{
+  "request_id": "a6a51003-2576-be0b-9a43-d3bdfeafc2f7",
+  "lease_id": "",
+  "lease_duration": 0,
+  "renewable": false,
+  "data": {
```
Comment (calvn) [inline website/source/api/system/sealwrap-rewrap.html.md]: We usually only include what's the `data` object in `Sample Response`s.

File: website/source/api/system/sealwrap-rewrap.html.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+The process will run in the background. Check the vault server logs for status
+and progress updates.
+
+| Method   | Path                   |
+| :------- | :--------------------- |
+| `POST`   | `/sys/sealwrap/rewrap` |
+
+The default status codes are:
+
+- `200` if a seal rewrap process is already running
```
Comment (calvn) [inline website/source/api/system/sealwrap-rewrap.html.md]: Does this return `200` along with what's usually returned by a `GET` request that contains info about the progress, or is this simply a 200 without any actual content?

File: website/source/api/system/sealwrap-rewrap.html.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+# `/sys/sealwrap/rewrap`
+
+~> **Enterprise Only** – These endpoints require Vault Enterprise.
+
+The `/sys/sealwrap/rewrap` endpoint is used to rewrap all seal wrapped entries.
+
+## Read Rewrap Status
+
+This endpoint retrieves if a seal rewrap process is currently running.
```
Comment (ncabatoff) [inline website/source/api/system/sealwrap-rewrap.html.md]: "retrieves if" sounds a bit clunky.  How about "reports whether"?

File: website/source/api/system/sealwrap-rewrap.html.md (hunk 2 above)
Comment (mgaffney) [reply website/source/api/system/sealwrap-rewrap.html.md]: Just `200`.

File: website/source/api/system/sealwrap-rewrap.html.md (hunk 1 above)
Comment (mgaffney) [reply website/source/api/system/sealwrap-rewrap.html.md]: I didn't know that. I will fix it. Thanks!

File: website/source/api/system/sealwrap-rewrap.html.md (hunk 3 above)
Comment (mgaffney) [reply website/source/api/system/sealwrap-rewrap.html.md]: Much better. Thanks!

File: website/source/api/system/sealwrap-rewrap.html.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+sidebar_current: "api-http-system-sealwrap-rewrap"
+description: |-
+  The `/sys/sealwrap/rewrap` endpoint is used to rewrap all seal wrapped entries.
+---
+
+# `/sys/sealwrap/rewrap`
+
+~> **Enterprise Only** – These endpoints require Vault Enterprise.
+
+The `/sys/sealwrap/rewrap` endpoint is used to rewrap all seal wrapped entries.
```
Comment (briankassouf) [inline website/source/api/system/sealwrap-rewrap.html.md]: Would be nice to include why someone might want to use a rewrap. e.g. after a seal migration, if you rotate the remote keyring and want to upgrade entries

Comment (mgaffney) [reply website/source/api/system/sealwrap-rewrap.html.md]: Added in #7689

**Pull request merged**